documentation/asciidoc/accessories/camera/libcamera_options_vid.adoc

```diff
@@ -0,0 +1,108 @@
+=== Video Command Line Options
+
+----
+	--quality,	-q		JPEG quality <number>
+----
+
+Set the JPEG quality. 100 is maximum quality and 50 is the default. Only applies when saving in MJPEG format.
+
+Example: `libcamera-vid --codec mjpeg -o test.mjpeg -q 80`
+
+----
+	--bitrate,	-b		H.264 bitrate <number>
+----
+
+Set the target bitrate for the H.264 encoder, in _bits per second_.
+
+Example: `libcamera-vid -b 10000000 --width 1920 --height 1080 -o test.h264`
+
+----
+	--intra,	-g		Intra-frame period (H.264 only) <number>
+----
+
+Sets the frequence of I (Intra) frames in the H.264 bitstream, as a number of frames. The default value is 60.
+
+Example: `libcamera-vid --intra 30 --width 1920 --height 1080 -o test.h264`
+
+----
+	--profile,			H.264 profile <string>
+----
+
+Set the H.264 profile. The value may be `baseline`, `main` or `high`.
+
+Example: `libcamera-vid --width 1920 --height 1080 --profile main -o test.h264`
+
+----
+	--level,			H.264 level <string>
+----
+
+Set the H.264 level. The value may be `4`, `4.1` or `4.2`.
+
+Example: `libcamera-vid --width 1920 --height 1080 --level 4.1 -o test.h264`
+
+
+
+
+----
+	--keypress,	-k		Capture image when Enter pressed
```

Comment:
Capture image -> Capture video?
(and the same also for the --signal option below)